improve serverless dms doc (#15793)
pingcap/docs: tidb-cloud/migrate-from-oracle-using-aws-dms.md

    ![Create AWS DMS Instance](/media/tidb-cloud/aws-dms-from-oracle-to-tidb-8.png)

> **Note:**
>

> > If you want a more thorough reference on creating AWS DMS replication instance to work with TiDB Serverless, refer to [Integrate TiDB Cloud with AWS DMS](/tidb-cloud/integrate-tidbcloud-with-aws-dms.md).

Reviewer: ```suggestion
> For detailed steps on creating an AWS DMS replication instance to work with TiDB Serverless, see [Integrate TiDB Cloud with AWS DMS](/tidb-cloud/integrate-tidbcloud-with-aws-dms.md).
```

    ![Create AWS DMS Target endpoint](/media/tidb-cloud/aws-dms-from-oracle-to-tidb-10.png)

> **Note:**
>

> > If you want a more thorough reference on creating TiDB Serverelss DMS endpoint, refer to [Integrate TiDB Cloud with AWS DMS](/tidb-cloud/integrate-tidbcloud-with-aws-dms.md).

Reviewer: ```suggestion
> For detailed steps on creating a TiDB Serverless DMS endpoint, see [Integrate TiDB Cloud with AWS DMS](/tidb-cloud/integrate-tidbcloud-with-aws-dms.md).
```
